docs/src/content/docs/getting-started/learning.md

```diff
@@ -5,90 +5,138 @@ sidebar:
   order: 2
 ---
 
-## Installation
+### Before we start
 
-First of all we will use @reatom/framework that incorparates many functions you may need in development.
+In the examples we will use the [@reatom/framework](/package/framework/) package which is an umbrella package for the most common reatom packages.
+You can install it with the command
 
 ```bash
 npm install --save @reatom/framework
 ```
 
-## Atoms are storing data and dependencies
+### Basic primitives
 
-Let's have trivial example that employs reatom's reactivity by implementing simple `a + b = c`:
+The reatom is based on three basic primitives: Atom, Action and Context
+Below you can see how they are used together, after which we will look at each line and what happens in it
 
-```typescript
-import { createCtx, atom } from '@reatom/framework'
+## Context
+
+The first thing we did was create a _context_. It is required for read, modify and subscribe operations. Later we will see what tricks it allows us to do, but now we will focus on the fact that it is enough to create **one context for the whole application**
 
+```ts
+import { createCtx } from '@reatom/framework'
 const ctx = createCtx()
+```
+
+## Atom
+
+### Create Atom
 
+```ts
+import { atom } from '@reatom/framework'
 const aAtom = atom(1, 'aAtom')
-const bAtom = atom(2, 'bAtom')
```

Comment:
Your have it deleted here but referenced in code below thus making it unusable with "copy and paste" learning process